static/_redirects

```diff
@@ -236,6 +236,7 @@
 /tools/terraform/reference/troubleshooting/private-access-error       https://aiven.io/docs/tools/terraform
 /tutorials/anomaly-detection                                          https://aiven.io/developer
 /valkey                                                               https://aiven.io/docs/products/valkey
+/platform/howto/manage-vpc-peering                         https://aiven.io/docs/platform/howto/manage-project-vpc
```

Comment:
Just a quick nitpick, could you select all these redirections and sort them alphabetically with vscode (using the sort feature). That helps maintain them and fix new 404s :)